src/redux/reducers/trendsStateByUser.js

```diff
@@ -47,8 +49,17 @@ const trendsStateByUser = (state = {}, action) => {
       return update(
         state,
         { [userId]: {
-          [FOCUSED_SLICE]: { $set: { slice, position } },
-          [FOCUSED_KEYS]: { $set: focusedKeys },
+          [FOCUSED_CBG_SLICE]: { $set: { slice, position } },
+          [FOCUSED_CBG_KEYS]: { $set: focusedKeys },
+        } }
+      );
+    }
+    case actionTypes.FOCUS_TRENDS_SMBG_RANGE_AVG: {
+      const { rangeAvgData: data, rangeAvgPosition: position, userId } = action.payload;
+      return update(
+        state,
+        { [userId]: {
+          [FOCUSED_SMBG_AVG]: { $set: { data, position } },
```

Comment:
I like that you've used the simple `data` for the name of the state tree field, but now I think where I used `slice` will be confusing. Can you go through and make the changes to make the cbg equivalent parallel - `data` instead of `slice`? (This will also mean updating the places where we _use_ this state from the tree, which is in both viz and blip. I can help you generate a list of places to modify if you don't want to just change it in the reducer and then play error whack-a-mole 😉)